### PR TITLE
Update Dependencies + add dependabot

### PR DESCRIPTION
**Pull request by ptbarnum4**
- [x] Update all dependencies and fix breaking changes
  - [REMOVED] `@chakra-ui/react@latest` 
  - [REMOVED] `@emotion/react@latest`
  - [REMOVED] `@emotion/styled@latest` 
  - [x] `@googleapis/calendar@latest`
  - [x] `@googleapis/sheets@latest`
  - [x] `@types/node@latest`
  - [x] `@types/react@latest`
  - [x] `@types/react-dom@latest`
  - [x] `@types/styled-components@latest`
  - [x] `@types/underscore@latest`
  - [x] `@types/validator@latest`
  - [x] `@typescript-eslint/eslint-plugin@latest`
  - [x] `@typescript-eslint/parser@latest`
  - [x] `axios@latest`
  - [x] `babel-plugin-styled-components@latest`
  - [x] `cli-color@latest`
  - [x] `cypress@latest`
  - [x] `eslint@latest`
  - [x] `eslint-config-next@latest`
  - [x] `framer-motion@latest`
  - [x] `google-auth-library@latest`
  - [x] `moment@latest`
  - [x] `moment-timezone@latest`
  - [x] `next@latest` #114 
  - [x] `react@latest` #114 
  - [x] `react-dom@latest` #114 
  - [x] `react-icons@latest`
  - [x] `rimraf@latest`
  - [x] `sharp@latest`
  - [Updated v5] `styled-components@latest`
  - [x] `typescript@latest`
  - [x] `underscore@latest`
  - [x] `validator@latest`
- [x] Add dependabot for future updates

### Dependencies to be updated
| Dependency                       | Old       | New        |
| -------------------------------- | --------- | ---------- |
| @chakra-ui/react                 | `1.8.8`   | `2.8.2`    |
| @emotion/react                   | `11.9.0`  | `11.11.1`  |
| @emotion/styled                  | `11.8.1`  | `11.11.0`  |
| @googleapis/calendar             | `2.0.0`   | `9.5.0`    |
| @googleapis/sheets               | `2.0.1`   | `5.0.5`    |
| @types/node                      | `17.0.34` | `20.10.5`  |
| @types/react                     | `17.0.45` | `18.2.45`  |
| @types/react-dom                 | `18.0.10` | `18.2.18`  |
| @types/styled-components         | `5.1.25`  | `5.1.34`   |
| @types/underscore                | `1.11.4`  | `1.11.15`  |
| @types/validator                 | `13.7.2`  | `13.11.7`  |
| @typescript-eslint/eslint-plugin | `5.25.0`  | `6.15.0`   |
| @typescript-eslint/parser        | `5.25.0`  | `6.15.0`   |
| axios                            | `0.26.1`  | `1.6.2`    |
| babel-plugin-styled-components   | `2.0.7`   | `2.1.4`    |
| cli-color                        | `2.0.2`   | `2.0.3`    |
| cypress                          | `9.6.1`   | `13.6.1`   |
| eslint                           | `8.15.0`  | `8.56.0`   |
| eslint-config-next               | `12.1.6`  | `14.0.4`   |
| framer-motion                    | `6.3.3`   | `10.16.16` |
| google-auth-library              | `8.0.2`   | `9.4.1`    |
| moment                           | `2.29.3`  | `2.29.4`   |
| moment-timezone                  | `0.5.34`  | `0.5.43`   |
| next                             | `12.1.6`  | `14.0.4`   |
| react                            | `17.0.2`  | `18.2.0`   |
| react-dom                        | `17.0.2`  | `18.2.0`   |
| react-icons                      | `4.3.1`   | `4.12.0`   |
| rimraf                           | `3.0.2`   | `5.0.5`    |
| sharp                            | `0.30.4`  | `0.33.1`   |
| styled-components                | `5.3.5`   | `6.1.1`    |
| typescript                       | `4.6.4`   | `5.3.3`    |
| underscore                       | `1.13.3`  | `1.13.6`   |
| validator                        | `13.7.0`  | `13.11.0`  |

---

### Updated button text to "Red Hat Display"
image

### Improve contrast in various places
image

### Major look/feel improvement to "Our History"
image

# Reset our history since everyone thought the parchment looked like garbage
Screenshot 2023-12-21 at 9 30 39 AM

# other changes -- request screenshots if interested

### CODE REVIEW

File: src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -93,11 +92,12 @@ export default function Nav({ alertInfo }: NavProps) {
 
   return (
     <NavbarStyles ref={navRef} animate={navAnimation} transition={navTransition}>
-      {OVERRIDE_NODE_ENV !== 'testing' ? null : (
+      {/* // TODO: Figure out why this breaks since update to nextjs v13 */}
```
Comment (harveysanders) [inline src/components/Navbar/Navbar.tsx:95]: What happens?

**Review by harveysanders — APPROVED** [review]
Beautiful

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (ptbarnum4) [reply src/components/Navbar/Navbar.tsx:95]: the page breaks

Comment (ptbarnum4) [reply src/components/Navbar/Navbar.tsx:95]: I'm pretty sure there is some weirs client/server thing happening because of the env variable

**Pull request merged**